spec/descriptions/tagSyntheticResults.md

```diff
@@ -166,6 +166,15 @@ Either tagFilters or tagFilterExpression can specify a custom property by its ke
 }]
 ```
 
+To filter tests by their active state (`true`/`false`) use the custom property label `synthetic.testActive`.
+```
+"tagFilters": [{ 
+  "name":"synthetic.testActive", 
+  "operator":"EQUALS",
+  "booleanValue": false 
+}]
+```
+
```

Comment:
synthetic.testActive is only applicable to getTestSummaryList API call, so this change and the change below need to go to the section: Get a list of Synthetic tests with Success Rate and Average Response Time data.